extension/duckdb/CMakeLists.txt

```diff
@@ -8,7 +8,10 @@ find_package(DuckDB REQUIRED)
 include_directories(
         ${PROJECT_SOURCE_DIR}/src/include
         src/include
-        ${DuckDB_INCLUDE_DIRS})
+        ${DuckDB_INCLUDE_DIRS}
+        ${PROJECT_SOURCE_DIR}/extension/httpfs/src/include)
```

Comment:
hmm Is it a good idea in one extension to include another extension's include directory?